node_frequency: 10
ip: "192.168.2.95"
port: 16171
